modules/ding_eresource/ding_eresource.pages_default.inc

```diff
@@ -267,6 +269,8 @@ function ding_eresource_default_page_manager_handlers() {
   $display->cache = array();
   $display->title = 'E-resurser';
   $display->uuid = 'd44ae923-a33f-4fc1-8d4c-2de4afe70ce0';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.